doc_build_aliases.sh

```diff
@@ -96,6 +96,9 @@ alias docbldecctl='$GIT_HOME/docs/build_docs --doc $GIT_HOME/ecctl/docs/index.as
 # Cloud - Elastic Cloud for K8s
 alias docbldk8s='$GIT_HOME/docs/build_docs --doc $GIT_HOME/cloud-on-k8s/docs/index.asciidoc --chunk 1'
 
+# Cloud - Terraform provider
+alias docbldtpec='$GIT_HOME/docs/build_docs --doc $GIT_HOME/terraform-provider-ec/docs/index.asciidoc --chunk 1'
```

Comment:
This link also depends on https://github.com/elastic/terraform-provider-ec/pull/163#pullrequestreview-522134387 :)

Reply:
Same here, I updated the build alias to use the new `docs-elastic` location.